GraphiQL

IlyaKozak opened the pull request:
1. Task requirements: https://github.com/rolling-scopes-school/tasks/blob/master/react/modules/graphiql.md

2.  Video Presentation:  

https://github.com/IlyaKozak/graphiql-app/assets/40767922/826d79ab-66b3-45c5-8a9b-156b7eba1f0f

3. Deploy: https://nextfireteam.netlify.app/

4. Submitted: 19.05.2023 / Deadline: 29.05.2023  

5.  **Cross-Check [90/90 points]**

**Welcome route [10 points]**
- [x] The welcome page contains general information about the developers, project, and course **+2 pts**  
- [x] In the upper right corner there are 2 buttons: Sign In and Sign Up **2 pts**  
- [x] If login token is valid and unexpired, Sign In and Sign Up buttons changed to the "Go to Main/GraphiQL" button **2 pts**  
- [x] When the token expires - the user is redirected to the "Welcome page" automatically **3 pts**  
- [x] Pressing the Sign In / Sign up button redirects a user to the route with the Sign In / Sign up form **1 pts**  

**Sign In / Sign Up [20 points]**
- [x] Buttons for Sign In / Sign Up / Sign Out are everywhere where they should be **5 pts**
- [x] Client-side validation is implemented **10 pts**
- [x] Upon successful login, the user is redirected to the Main/GraphiQL page **3 pts**
- [x] If the user is already logged in and tries to reach these routes, he/she is redirected to the Main/GraphiQL page **2 pts**

**GraphiQL route [50 points]**
- [x] Working editor allowing to edit the query **15 pts**
- [x] Working documentation explorer (visible only when SDL request is successful) **15 pts**
- [x] Variables section (can be closed/opened) **5 pts**
- [x] Headers section (can be closed/opened) **5 pts**
- [x] Response section **10 pts**

**General requirements [10 points]**
- [x] Localization (en/ru) **5 pts**
- [x] Sticky (animated) header **5 pts**

IlyaKozak committed:
build: add firebase

Bulrock committed:
feat: implement return Schema object for endpoints filled with input

Bulrock committed:
refactor: remove constant from GraphiQLService in separate folder, fix typo in locales

Bulrock committed:
Merge pull request #31 from IlyaKozak/maxvalazhynsky-develop

feat: implement return Schema object for endpoints filled with input

Bulrock committed:
feat: add responsive styles for MainHeader component

Bulrock committed:
feat: implement Response and Editor sections

Bulrock committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into maxvalazhynsky-develop

Bulrock committed:
update: pull changes from develop and install codemirror

Bulrock committed:
Merge pull request #32 from IlyaKozak/maxvalazhynsky-develop

feat: implement Editor and Response sections

Siaw4ik committed:
feat: implement domentation explorer 1 step

Siaw4ik committed:
update: pull changes from develop and resolve conflict

Siaw4ik committed:
fix: linter warnings and change type

Siaw4ik committed:
refactor: create constant for default graphQL endpoint

Siaw4ik committed:
Merge pull request #33 from IlyaKozak/siaw4ik-develop

feat: add base implementation of docs explorer

Bulrock committed:
feat: install types for codemirror

Bulrock committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into maxvalazhynsky-develop

IlyaKozak committed:
feat: add firebase auth

IlyaKozak committed:
merge: with develop

IlyaKozak committed:
feat: add auth page signin & signup

IlyaKozak committed:
feat: add automatic redirect from auth to graphql page if user is authorized

Bulrock committed:
feat: implement responsive styles for Response, Editor and Variables sections of the playground

Bulrock committed:
feat: implement sidebar for DOCS

Bulrock committed:
feat: implement cursor pointer for DOCS label

IlyaKozak committed:
Merge pull request #34 from IlyaKozak/ilyakozak-develop

feat: add firebase auth

Bulrock committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into maxvalazhynsky-develop

Bulrock committed:
refactor: install dependencies

Bulrock committed:
Merge pull request #35 from IlyaKozak/maxvalazhynsky-develop

feat: implement responsive styles for graphiQl playground

Bulrock committed:
feat: add endpoint section to the main page, add logo and signout button

Bulrock committed:
feat: implement sticky header for the main page

Bulrock committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into maxvalazhynsky-develop

Bulrock committed:
Merge pull request #36 from IlyaKozak/maxvalazhynsky-develop

feat: add endpoint section to the main page, add logo and signout button and implement sticky header

Siaw4ik committed:
feat: add schema for all types without arguments

Siaw4ik committed:
update: pull changes from develop

Siaw4ik committed:
fix: change style name

Siaw4ik committed:
Merge pull request #38 from IlyaKozak/siaw4ik-develop

feat: add schema for all types without arguments

Bulrock committed:
feat: add headers section, update responsive styles

Bulrock committed:
feat: add SwitchLocale component to the MainHeader component

Bulrock committed:
feat: add locales for the Main page

Bulrock committed:
refactor: placeholders font styles

Bulrock committed:
refactor: update lables on hover effect styles

Bulrock committed:
refactor: modify GraphiQLService

Siaw4ik committed:
feat: add type modifiers to documentation explorer

Siaw4ik committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into siaw4ik-develop

Siaw4ik committed:
refactor: remove comment

Siaw4ik committed:
refactor: linter fixes

Bulrock committed:
feat: implement variables for graphiQL request

Siaw4ik committed:
Merge pull request #39 from IlyaKozak/siaw4ik-develop

feat: add type modifiers to documentation explorer

Bulrock committed:
feat: implement headers for graphiQL request

Bulrock committed:
update: pull changes from develop

Bulrock committed:
feat: add paddings for DOCS explorer window

Bulrock committed:
fix: remove unused properties from DocsType

Bulrock committed:
fix: update eslint config and fix warnings

Bulrock committed:
Merge pull request #40 from IlyaKozak/maxvalazhynsky-develop

feat: implement headers and variables sections for the playground graphQL requests

Siaw4ik committed:
feat: implement arguments base names

Siaw4ik committed:
update: pull changes from develop

Siaw4ik committed:
Merge pull request #41 from IlyaKozak/siaw4ik-develop

feat: implement arguments base names

Bulrock committed:
feat: implement errors handling

Bulrock committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into maxvalazhynsky-develop

IlyaKozak committed:
feat: add loader component

IlyaKozak committed:
feat: add main header to welcome and auth pages

IlyaKozak committed:
feat: change next config to server static assets

IlyaKozak committed:
Merge pull request #42 from IlyaKozak/ilyakozak-develop

feat: add main header to welcome and auth pages & loader component styles

Bulrock committed:
fix: lable translation and header z-index

Bulrock committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into maxvalazhynsky-develop

Bulrock committed:
refactor: change the indents in the editor

Bulrock committed:
Merge pull request #43 from IlyaKozak/maxvalazhynsky-develop

feat: implement errors handling

IlyaKozak committed:
feat: add email & password regex validation

IlyaKozak committed:
feat: add form validation to email & password

IlyaKozak committed:
fix: setValidationFields logic in Auth component

IlyaKozak committed:
fix: reset error messages in auth component if switching from signup to signin form or reverse

IlyaKozak committed:
style: center loader in auth component

IlyaKozak committed:
Merge pull request #44 from IlyaKozak/ilyakozak-develop

feat: add form validation to email & password

Siaw4ik committed:
feat: add type modifiers argumets to documentation explorer and add cross image for close documentation

Siaw4ik committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into siaw4ik-develop

Siaw4ik committed:
refactor: remove comment

Siaw4ik committed:
Merge pull request #46 from IlyaKozak/siaw4ik-develop

feat: add type modifiers argumets to documentation explorer and add c…

Bulrock committed:
refactor lables styles

Bulrock committed:
update: pull changes from develop and resolve conflicts

Bulrock committed:
update: pull changes from develop and resolve conflicts

Bulrock committed:
refactor: lables styles and docs styles

Bulrock committed:
feat: implement header animation on scroll

Bulrock committed:
refactor: update fonts styles for the main page

Bulrock committed:
refactor: update paddings for the response section

Siaw4ik committed:
feat: add transitions to argument types and add style elements displaying field type

Bulrock committed:
refactor: update error handling

Siaw4ik committed:
Merge pull request #47 from IlyaKozak/siaw4ik-develop

feat: add transitions to argument types and add style elements displa…

Bulrock committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into maxvalazhynsky-develop

Bulrock committed:
fix: window error on the server side

Bulrock committed:
refactor: lables paddings

Bulrock committed:
refactore: move font style to the global.css

Bulrock committed:
Merge pull request #48 from IlyaKozak/maxvalazhynsky-develop

feat: implement header animation on scroll on the main page, update errors handling, update styles

Bulrock committed:
refactor: lable and buttons styles

Siaw4ik committed:
feat: add field types and arguments as a separate part when clicking on the object key

Siaw4ik committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into siaw4ik-develop

Siaw4ik committed:
Merge pull request #49 from IlyaKozak/siaw4ik-develop

feat: add field types and arguments as a separate part when clicking on the object key

Bulrock committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into maxvalazhynsky-develop

Siaw4ik committed:
fix: add scroll, remove header when there are no arguments, fix description redrawing and add argument type rendering

Siaw4ik committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into siaw4ik-develop

IlyaKozak committed:
feat: disable button on loading in auth component

Siaw4ik committed:
fix: fix disappearing label in docs

Siaw4ik committed:
Merge pull request #50 from IlyaKozak/siaw4ik-develop

fix: add scroll, remove header when there are no arguments, fix description redrawing and add argument type rendering

Siaw4ik committed:
refactor: split the existing long component into its component parts by removing repeated lines of code and forming smaller components

Siaw4ik committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into siaw4ik-develop

Siaw4ik committed:
Merge pull request #51 from IlyaKozak/siaw4ik-develop

refactor: split the existing long component into its component parts by removing repeated lines of code and forming smaller components

Bulrock committed:
feat: implement hiding variables and headers sections with arrow click

Bulrock committed:
refactor: responsive styles for main page

Bulrock committed:
feat: implement toggle variablesHeaderWrapper section classes withclick on the header and variables lables

Bulrock committed:
fix: main page responsive styles

Bulrock committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into maxvalazhynsky-develop

Bulrock committed:
Merge pull request #55 from IlyaKozak/maxvalazhynsky-develop

feat: implement hiding variables and headers sections with arrow click

Bulrock committed:
refactor: separate query, variables, headers textereas from single EditorSection component to separate MyTextarea component

Bulrock committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into maxvalazhynsky-develop

IlyaKozak committed:
feat: add custom 404 page

IlyaKozak committed:
feat: add ru localization for firebase auth errors

IlyaKozak committed:
feat: add go home link to custom 404 page

Bulrock committed:
Merge pull request #56 from IlyaKozak/maxvalazhynsky-develop

refactor: separate query, variables, headers textereas from single EditorSection component to separate MyTextarea component

IlyaKozak committed:
Merge pull request #57 from IlyaKozak/ilyakozak-develop

feat: disable button on loading in auth component / custom 404 & firebase error ru localization

Siaw4ik committed:
feat: implement lazy-loading for docs and refactor buttons styles and remove scroll when toggle languages

Siaw4ik committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into siaw4ik-develop

Siaw4ik committed:
fix: remove comment

IlyaKozak committed:
feat: add textarea text formatting for enter, tab, backspace, open bracket

Siaw4ik committed:
feat: implement loader on change endpoint button click

Siaw4ik committed:
refactor: lazy-loader logic

Siaw4ik committed:
Merge pull request #58 from IlyaKozak/siaw4ik-develop

feat: implement lazy-loading for docs and refactor buttons styles and remove scroll when toggle languages

Siaw4ik committed:
fix: linter erros

Siaw4ik committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into siaw4ik-develop

Siaw4ik committed:
Merge pull request #59 from IlyaKozak/siaw4ik-develop

fix: linter erros

Siaw4ik committed:
feat: localization for docs

Siaw4ik committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into siaw4ik-develop

Bulrock committed:
feat: add footer, refactor header, update responsive styles

Bulrock committed:
feat: implement endpoint section message translation, update message styles

Bulrock committed:
fix: problem with absence of the window object on the server side

Siaw4ik committed:
fix: a mistake in the word descrAbsence

Siaw4ik committed:
Merge pull request #60 from IlyaKozak/siaw4ik-develop

feat: localization for docs

Bulrock committed:
refactor: add constants for resolutions and dimensions

Bulrock committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into maxvalazhynsky-develop

Siaw4ik committed:
feat: add loader on request submit

Siaw4ik committed:
fix: font saturation in docs with russian localization

Siaw4ik committed:
feat: add style of mobile for loader on request submit

Siaw4ik committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into siaw4ik-develop

Bulrock committed:
fix: responsive style bugs for 430px window width

Siaw4ik committed:
Merge pull request #63 from IlyaKozak/siaw4ik-develop

feat: add loader on request submit and fix: font saturation in docs with russian localization

Bulrock committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into maxvalazhynsky-develop

Bulrock committed:
Merge pull request #64 from IlyaKozak/maxvalazhynsky-develop

feat: add footer, refactor header, update and fix responsive styles

Bulrock committed:
refactor: change default style, change RSSchool link

Bulrock committed:
refactor: font styles and responsive styles

Bulrock committed:
refactor: auth page, SignUp and SignIn forms styles

Bulrock committed:
feat: implement responsive styles for the Welcome page

Bulrock committed:
refactor: Welcome page responsive styles

Bulrock committed:
feat: add technology stack section to the Welcome page

Bulrock committed:
feat: implement translation for the stack section

IlyaKozak committed:
feat: add initial hints for query editor section

IlyaKozak committed:
merge: develop

IlyaKozak committed:
Merge pull request #65 from IlyaKozak/ilyakozak-develop

feat: add initial hints for query editor section + formatting

Bulrock committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into maxvalazhynsky-develop

Bulrock committed:
Merge pull request #66 from IlyaKozak/maxvalazhynsky-develop

refactor: auth page, SignUp and SignIn forms styles, implement responsive styles for the Welcome page

IlyaKozak committed:
fix: section clearing after endpoint change

Bulrock committed:
feat: implement auth page background image

Bulrock committed:
refactor: welcome page title translation

IlyaKozak committed:
feat: add hints for args in editor section

Bulrock committed:
implement: reusable and configurable button with link inside

Bulrock committed:
refactor:replace with button 404 page home link

IlyaKozak committed:
feat: add response section highlighting

IlyaKozak committed:
Merge pull request #71 from IlyaKozak/ilyakozak-develop

feat: add response section highlighting

Bulrock committed:
feat: implement base styles for welcome page without about-us section

Bulrock committed:
refactor: update responsive styles

Bulrock committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into maxvalazhynsky-develop

Bulrock committed:
Merge pull request #72 from IlyaKozak/maxvalazhynsky-develop

feat: implement responsive styles for welcome page without about-us section, implement responsive styles for auth page

Bulrock committed:
fix: background absence on the server side

Bulrock committed:
Merge pull request #73 from IlyaKozak/maxvalazhynsky-develop

fix: background absence on the server side

Siaw4ik committed:
feat: implement error handling and add Toast for errors

Siaw4ik committed:
update: pull changes from develop

Siaw4ik committed:
Merge pull request #74 from IlyaKozak/siaw4ik-develop

feat: implement error handling and add Toast for errors

Bulrock committed:
feat: implement about section of the welcome page

Bulrock committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into maxvalazhynsky-develop

Bulrock committed:
refactor: update logoIcon responsive style

Bulrock committed:
Merge pull request #76 from IlyaKozak/maxvalazhynsky-develop

feat: implement about section of the welcome page

Bulrock committed:
refactor: add logos animations pause on welcome page logos hover

Bulrock committed:
Merge branch 'develop' of https://github.com/IlyaKozak/graphiql-app into maxvalazhynsky-develop

Bulrock committed:
refactor: remove unused comments, replace tabs with spaces

Bulrock committed:
refactor: endpoint button responsive style

Bulrock committed:
Merge pull request #77 from IlyaKozak/maxvalazhynsky-develop

refactor: add logos animations pause on welcome page logos hover

IlyaKozak committed:
style: minor corrections

IlyaKozak committed:
feat: add select hints by space and mouse click

IlyaKozak committed:
build: install sharp package strongly recommended by next.js for production image optimization

IlyaKozak committed:
docs: update readme

IlyaKozak committed:
feat: add error boundary

IlyaKozak committed:
Merge pull request #79 from IlyaKozak/ilyakozak-develop

feat: error boundary

IlyaKozak committed:
fix: console warning preloaded with link preload was not used within a few seconds

IlyaKozak committed:
Merge pull request #80 from IlyaKozak/ilyakozak-develop

fix: console warning preloaded with link preload was not used within …

IlyaKozak committed:
fix: big arrow on mobile screen

IlyaKozak committed:
Merge pull request #81 from IlyaKozak/ilyakozak-develop

fix: big arrow on mobile screen

Siaw4ik committed:
fix: the appearance of the schema documentation window with the wrong schema url

Siaw4ik committed:
Merge pull request #83 from IlyaKozak/siaw4ik-develop

fix: the appearance of the schema documentation window with the wrong…

Skave-a commented on src/components/Auth/Auth.tsx:
```diff
@@ -0,0 +1,195 @@
+              <>{signUp}</>
+            ) : (
+              <>{signIn}</>
+            )}
```
Слишком большое условие в рендере, необходимо вывести в отдельную функцию.
Извлечение вложенной троичной операции в независимую функцию улучшит читабельность кода и упростит дальнейшую работу с ним, также снизит вероятность ошибок при изменении

IlyaKozak replied:
refactored (da7d334149597106c3f3f2b2af075609c5691800)

Skave-a commented on src/components/Docs/Docs.tsx:
```diff
@@ -0,0 +1,139 @@
+
+import { DocsType } from '@/types/docs';
+import { __Field, __Type, __InputValue } from '@/types/schema';
+import { ValueRoot } from '@/types/docs';
```
Несколько импортов из одного и того же модуля должны быть объединены вместе, чтобы улучшить читаемость.
строки 3 и 5 можно объединить

IlyaKozak replied:
done (e6f720c68fd779a18f00480306f7e68e9571cd5a)

Skave-a commented on src/components/EditorSection/EditorSection.tsx:
```diff
@@ -0,0 +1,229 @@
+            setShowToast(true);
+            setErrorMessageToast(error.message);
+            setResponse('');
+          } else {
```
один и тот же код  - строки 89-91 и 93-95

IlyaKozak replied:
refactored (7662dd8f7f611b2e13fdd3fcfceabb9ccb7f17e7)

Skave-a commented on src/components/Footer/Footer.module.css:
```diff
@@ -0,0 +1,59 @@
+  transition: all .4s ease-in-out;
+  background-color: unset;
+}
+
```
можно объединить .logoRS с 30 строки по 42

IlyaKozak replied:
done (8aa7dcf7080218e1590d4a071e3f32196ef46bff)

Skave-a commented on src/components/MyTextarea/HintsModal.tsx:
```diff
@@ -0,0 +1,88 @@
+          : hints.length < MIN_NUMBER_OF_OPTIONS
+          ? MIN_NUMBER_OF_OPTIONS
+          : hints.length
+      }
```
Лучше никогда не использовать троичный тернарный оператор, оно работает, но читать это тяжело

IlyaKozak replied:
refactored (da0e9401cec4b080d7b6e8fb9a37420280b39dfc)

Skave-a commented on src/context/auth.context.tsx:
```diff
@@ -0,0 +1,83 @@
+    createUser: firebaseAuth ? createUserWithEmailAndPassword.bind(null, firebaseAuth) : null,
+    signInUser: firebaseAuth ? signInWithEmailAndPassword.bind(null, firebaseAuth) : null,
+    signOutUser: firebaseAuth ? signOut.bind(null, firebaseAuth) : null,
+  };
```
Объект authContextValue изменяется при каждом рендеринге. Чтобы исправить это, подумайте о том, чтобы обернуть его в хук useMemo.

IlyaKozak replied:
done (659f9f8e29ae3c260f76744154e36ddae296a049)

Skave-a commented on src/services/findNameType.ts:
```diff
@@ -0,0 +1,24 @@
+  if (obj.name !== null && obj.ofType === null) {
+    const keyName = obj.name;
+    let valueName = obj.name;
+    arr.reverse().forEach((item) => {
```
нельзя менять пропсы в дочернем элементе, используя функцию reverse(), вы изменяете изначальный массив

IlyaKozak replied:
It's not a React component, so it's not forbidden to change function arguments. This function works with deeply nested objects of graphql schema, calls itself recursively. In general, I agree that it is better to write pure functions.

Skave-a commented on src/types/docs.ts:
```diff
@@ -0,0 +1,64 @@
+import { __Schema as Schema } from './schema';
+import { __Type, __TypeKind, __EnumValue, __Field, __InputValue } from './schema';
+
```
Несколько импортов из одного и того же модуля должны быть объединены вместе, чтобы улучшить читаемость.

IlyaKozak replied:
done (e6f720c68fd779a18f00480306f7e68e9571cd5a)

Skave-a commented on src/utils/validation.ts:
```diff
@@ -0,0 +1,11 @@
+  return EMAIL_REGEX.test(email);
+};
+
+const PASSWORD_REGEX = /^(?=.*\d)(?=.*[a-z])(?=.*[^a-zA-Z0-9])(?!.*\s).{8,}$/i;
```
a-z дублируется

IlyaKozak replied:
refactored (ffed61c00858e2d8a5fa24434cb3002c92d163c9)

Skave-a commented on src/components/ResponseSection/ResponseSection.module.css:
```diff
@@ -0,0 +1,43 @@
+  min-width: 25vw;
+  width: 50vw;
+  max-width: 75vw;
+  width: 100%;
```
width указан дважды

IlyaKozak replied:
done (8aa7dcf7080218e1590d4a071e3f32196ef46bff)

dependabot[bot] committed:
build(deps): bump sharp from 0.32.1 to 0.32.6

Bumps [sharp](https://github.com/lovell/sharp) from 0.32.1 to 0.32.6.
- [Release notes](https://github.com/lovell/sharp/releases)
- [Changelog](https://github.com/lovell/sharp/blob/main/docs/changelog.md)
- [Commits](https://github.com/lovell/sharp/compare/v0.32.1...v0.32.6)

---
updated-dependencies:
- dependency-name: sharp
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #84 from IlyaKozak/dependabot/npm_and_yarn/sharp-0.32.6

build(deps): bump sharp from 0.32.1 to 0.32.6

dependabot[bot] committed:
build(deps): bump protobufjs from 6.11.3 to 6.11.4

Bumps [protobufjs](https://github.com/protobufjs/protobuf.js) from 6.11.3 to 6.11.4.
- [Release notes](https://github.com/protobufjs/protobuf.js/releases)
- [Changelog](https://github.com/protobufjs/protobuf.js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/protobufjs/protobuf.js/commits)

---
updated-dependencies:
- dependency-name: protobufjs
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #85 from IlyaKozak/dependabot/npm_and_yarn/protobufjs-6.11.4

build(deps): bump protobufjs from 6.11.3 to 6.11.4

dependabot[bot] committed:
build(deps): bump postcss and next

Bumps [postcss](https://github.com/postcss/postcss) to 8.4.31 and updates ancestor dependency [next](https://github.com/vercel/next.js). These dependencies need to be updated together.


Updates `postcss` from 8.4.14 to 8.4.31
- [Release notes](https://github.com/postcss/postcss/releases)
- [Changelog](https://github.com/postcss/postcss/blob/main/CHANGELOG.md)
- [Commits](https://github.com/postcss/postcss/compare/8.4.14...8.4.31)

Updates `next` from 13.3.1 to 13.5.6
- [Release notes](https://github.com/vercel/next.js/releases)
- [Changelog](https://github.com/vercel/next.js/blob/canary/release.js)
- [Commits](https://github.com/vercel/next.js/compare/v13.3.1...v13.5.6)

---
updated-dependencies:
- dependency-name: postcss
  dependency-type: indirect
- dependency-name: next
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #86 from IlyaKozak/dependabot/npm_and_yarn/postcss-and-next-8.4.31

build(deps): bump postcss and next

dependabot[bot] committed:
build(deps): bump word-wrap from 1.2.3 to 1.2.5

Bumps [word-wrap](https://github.com/jonschlinkert/word-wrap) from 1.2.3 to 1.2.5.
- [Release notes](https://github.com/jonschlinkert/word-wrap/releases)
- [Commits](https://github.com/jonschlinkert/word-wrap/compare/1.2.3...1.2.5)

---
updated-dependencies:
- dependency-name: word-wrap
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #87 from IlyaKozak/dependabot/npm_and_yarn/word-wrap-1.2.5

build(deps): bump word-wrap from 1.2.3 to 1.2.5

dependabot[bot] committed:
build(deps): bump semver from 6.3.0 to 6.3.1

Bumps [semver](https://github.com/npm/node-semver) from 6.3.0 to 6.3.1.
- [Release notes](https://github.com/npm/node-semver/releases)
- [Changelog](https://github.com/npm/node-semver/blob/v6.3.1/CHANGELOG.md)
- [Commits](https://github.com/npm/node-semver/compare/v6.3.0...v6.3.1)

---
updated-dependencies:
- dependency-name: semver
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #88 from IlyaKozak/dependabot/npm_and_yarn/semver-6.3.1

build(deps): bump semver from 6.3.0 to 6.3.1

dependabot[bot] committed:
build(deps): bump braces from 3.0.2 to 3.0.3

Bumps [braces](https://github.com/micromatch/braces) from 3.0.2 to 3.0.3.
- [Changelog](https://github.com/micromatch/braces/blob/master/CHANGELOG.md)
- [Commits](https://github.com/micromatch/braces/compare/3.0.2...3.0.3)

---
updated-dependencies:
- dependency-name: braces
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #89 from IlyaKozak/dependabot/npm_and_yarn/braces-3.0.3

build(deps): bump braces from 3.0.2 to 3.0.3

dependabot[bot] committed:
build(deps): bump micromatch from 4.0.5 to 4.0.8

Bumps [micromatch](https://github.com/micromatch/micromatch) from 4.0.5 to 4.0.8.
- [Release notes](https://github.com/micromatch/micromatch/releases)
- [Changelog](https://github.com/micromatch/micromatch/blob/master/CHANGELOG.md)
- [Commits](https://github.com/micromatch/micromatch/compare/4.0.5...4.0.8)

---
updated-dependencies:
- dependency-name: micromatch
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #90 from IlyaKozak/dependabot/npm_and_yarn/micromatch-4.0.8

build(deps): bump micromatch from 4.0.5 to 4.0.8

dependabot[bot] committed:
build(deps): bump next from 13.5.6 to 14.1.1

Bumps [next](https://github.com/vercel/next.js) from 13.5.6 to 14.1.1.
- [Release notes](https://github.com/vercel/next.js/releases)
- [Changelog](https://github.com/vercel/next.js/blob/canary/release.js)
- [Commits](https://github.com/vercel/next.js/compare/v13.5.6...v14.1.1)

---
updated-dependencies:
- dependency-name: next
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #91 from IlyaKozak/dependabot/npm_and_yarn/next-14.1.1

build(deps): bump next from 13.5.6 to 14.1.1

dependabot[bot] committed:
build(deps): bump @grpc/grpc-js and firebase

Bumps [@grpc/grpc-js](https://github.com/grpc/grpc-node) to 1.9.15 and updates ancestor dependency [firebase](https://github.com/firebase/firebase-js-sdk). These dependencies need to be updated together.


Updates `@grpc/grpc-js` from 1.7.3 to 1.9.15
- [Release notes](https://github.com/grpc/grpc-node/releases)
- [Commits](https://github.com/grpc/grpc-node/compare/@grpc/grpc-js@1.7.3...@grpc/grpc-js@1.9.15)

Updates `firebase` from 9.20.0 to 10.13.1
- [Release notes](https://github.com/firebase/firebase-js-sdk/releases)
- [Changelog](https://github.com/firebase/firebase-js-sdk/blob/main/CHANGELOG.md)
- [Commits](https://github.com/firebase/firebase-js-sdk/compare/firebase@9.20.0...firebase@10.13.1)

---
updated-dependencies:
- dependency-name: "@grpc/grpc-js"
  dependency-type: indirect
- dependency-name: firebase
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #92 from IlyaKozak/dependabot/npm_and_yarn/multi-9d2bb77314

build(deps): bump @grpc/grpc-js and firebase

dependabot[bot] committed:
build(deps): bump next from 14.1.1 to 14.2.10

Bumps [next](https://github.com/vercel/next.js) from 14.1.1 to 14.2.10.
- [Release notes](https://github.com/vercel/next.js/releases)
- [Changelog](https://github.com/vercel/next.js/blob/canary/release.js)
- [Commits](https://github.com/vercel/next.js/compare/v14.1.1...v14.2.10)

---
updated-dependencies:
- dependency-name: next
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #93 from IlyaKozak/dependabot/npm_and_yarn/next-14.2.10

build(deps): bump next from 14.1.1 to 14.2.10

dependabot[bot] committed:
build(deps): bump cross-spawn from 7.0.3 to 7.0.6

Bumps [cross-spawn](https://github.com/moxystudio/node-cross-spawn) from 7.0.3 to 7.0.6.
- [Changelog](https://github.com/moxystudio/node-cross-spawn/blob/master/CHANGELOG.md)
- [Commits](https://github.com/moxystudio/node-cross-spawn/compare/v7.0.3...v7.0.6)

---
updated-dependencies:
- dependency-name: cross-spawn
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #94 from IlyaKozak/dependabot/npm_and_yarn/cross-spawn-7.0.6

build(deps): bump cross-spawn from 7.0.3 to 7.0.6

dependabot[bot] committed:
build(deps): bump nanoid from 3.3.7 to 3.3.8

Bumps [nanoid](https://github.com/ai/nanoid) from 3.3.7 to 3.3.8.
- [Release notes](https://github.com/ai/nanoid/releases)
- [Changelog](https://github.com/ai/nanoid/blob/main/CHANGELOG.md)
- [Commits](https://github.com/ai/nanoid/compare/3.3.7...3.3.8)

---
updated-dependencies:
- dependency-name: nanoid
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #95 from IlyaKozak/dependabot/npm_and_yarn/nanoid-3.3.8

build(deps): bump nanoid from 3.3.7 to 3.3.8

dependabot[bot] committed:
build(deps): bump next from 14.2.10 to 14.2.15

Bumps [next](https://github.com/vercel/next.js) from 14.2.10 to 14.2.15.
- [Release notes](https://github.com/vercel/next.js/releases)
- [Changelog](https://github.com/vercel/next.js/blob/canary/release.js)
- [Commits](https://github.com/vercel/next.js/compare/v14.2.10...v14.2.15)

---
updated-dependencies:
- dependency-name: next
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #96 from IlyaKozak/dependabot/npm_and_yarn/next-14.2.15

build(deps): bump next from 14.2.10 to 14.2.15

dependabot[bot] committed:
build(deps): bump next from 14.2.15 to 14.2.21

Bumps [next](https://github.com/vercel/next.js) from 14.2.15 to 14.2.21.
- [Release notes](https://github.com/vercel/next.js/releases)
- [Changelog](https://github.com/vercel/next.js/blob/canary/release.js)
- [Commits](https://github.com/vercel/next.js/compare/v14.2.15...v14.2.21)

---
updated-dependencies:
- dependency-name: next
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #97 from IlyaKozak/dependabot/npm_and_yarn/next-14.2.21

build(deps): bump next from 14.2.15 to 14.2.21

dependabot[bot] committed:
build(deps): bump undici and firebase

Removes [undici](https://github.com/nodejs/undici). It's no longer used after updating ancestor dependency [firebase](https://github.com/firebase/firebase-js-sdk). These dependencies need to be updated together.


Removes `undici`

Updates `firebase` from 10.13.1 to 11.2.0
- [Release notes](https://github.com/firebase/firebase-js-sdk/releases)
- [Changelog](https://github.com/firebase/firebase-js-sdk/blob/main/CHANGELOG.md)
- [Commits](https://github.com/firebase/firebase-js-sdk/compare/firebase@10.13.1...firebase@11.2.0)

---
updated-dependencies:
- dependency-name: undici
  dependency-type: indirect
- dependency-name: firebase
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #98 from IlyaKozak/dependabot/npm_and_yarn/multi-5f1fbf40b3

build(deps): bump undici and firebase

dependabot[bot] committed:
build(deps): bump next from 14.2.21 to 14.2.30

Bumps [next](https://github.com/vercel/next.js) from 14.2.21 to 14.2.30.
- [Release notes](https://github.com/vercel/next.js/releases)
- [Changelog](https://github.com/vercel/next.js/blob/canary/release.js)
- [Commits](https://github.com/vercel/next.js/compare/v14.2.21...v14.2.30)

---
updated-dependencies:
- dependency-name: next
  dependency-version: 14.2.30
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #99 from IlyaKozak/dependabot/npm_and_yarn/next-14.2.30

build(deps): bump next from 14.2.21 to 14.2.30

dependabot[bot] committed:
build(deps): bump @babel/runtime from 7.21.0 to 7.27.6

Bumps [@babel/runtime](https://github.com/babel/babel/tree/HEAD/packages/babel-runtime) from 7.21.0 to 7.27.6.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.27.6/packages/babel-runtime)

---
updated-dependencies:
- dependency-name: "@babel/runtime"
  dependency-version: 7.27.6
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #100 from IlyaKozak/dependabot/npm_and_yarn/babel/runtime-7.27.6

build(deps): bump @babel/runtime from 7.21.0 to 7.27.6

dependabot[bot] committed:
build(deps): bump brace-expansion from 1.1.11 to 1.1.12

Bumps [brace-expansion](https://github.com/juliangruber/brace-expansion) from 1.1.11 to 1.1.12.
- [Release notes](https://github.com/juliangruber/brace-expansion/releases)
- [Commits](https://github.com/juliangruber/brace-expansion/compare/1.1.11...v1.1.12)

---
updated-dependencies:
- dependency-name: brace-expansion
  dependency-version: 1.1.12
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #101 from IlyaKozak/dependabot/npm_and_yarn/brace-expansion-1.1.12

build(deps): bump brace-expansion from 1.1.11 to 1.1.12

dependabot[bot] committed:
build(deps): bump tar-fs

Bumps  and [tar-fs](https://github.com/mafintosh/tar-fs). These dependencies needed to be updated together.

Updates `tar-fs` from 2.1.1 to 2.1.3
- [Commits](https://github.com/mafintosh/tar-fs/compare/v2.1.1...v2.1.3)

Updates `tar-fs` from 3.0.4 to 3.1.0
- [Commits](https://github.com/mafintosh/tar-fs/compare/v2.1.1...v2.1.3)

---
updated-dependencies:
- dependency-name: tar-fs
  dependency-version: 2.1.3
  dependency-type: indirect
- dependency-name: tar-fs
  dependency-version: 3.1.0
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #102 from IlyaKozak/dependabot/npm_and_yarn/multi-fe18226ecc

build(deps): bump tar-fs

dependabot[bot] committed:
build(deps): bump next from 14.2.30 to 14.2.32

Bumps [next](https://github.com/vercel/next.js) from 14.2.30 to 14.2.32.
- [Release notes](https://github.com/vercel/next.js/releases)
- [Changelog](https://github.com/vercel/next.js/blob/canary/release.js)
- [Commits](https://github.com/vercel/next.js/compare/v14.2.30...v14.2.32)

---
updated-dependencies:
- dependency-name: next
  dependency-version: 14.2.32
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #103 from IlyaKozak/dependabot/npm_and_yarn/next-14.2.32

build(deps): bump next from 14.2.30 to 14.2.32

dependabot[bot] committed:
build(deps): bump tar-fs

Bumps  and [tar-fs](https://github.com/mafintosh/tar-fs). These dependencies needed to be updated together.

Updates `tar-fs` from 2.1.3 to 2.1.4
- [Commits](https://github.com/mafintosh/tar-fs/compare/v2.1.3...v2.1.4)

Updates `tar-fs` from 3.1.0 to 3.1.1
- [Commits](https://github.com/mafintosh/tar-fs/compare/v2.1.3...v2.1.4)

---
updated-dependencies:
- dependency-name: tar-fs
  dependency-version: 2.1.4
  dependency-type: indirect
- dependency-name: tar-fs
  dependency-version: 3.1.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #104 from IlyaKozak/dependabot/npm_and_yarn/multi-c30bf1d5d2

build(deps): bump tar-fs

dependabot[bot] committed:
build(deps): bump js-yaml from 4.1.0 to 4.1.1

Bumps [js-yaml](https://github.com/nodeca/js-yaml) from 4.1.0 to 4.1.1.
- [Changelog](https://github.com/nodeca/js-yaml/blob/master/CHANGELOG.md)
- [Commits](https://github.com/nodeca/js-yaml/compare/4.1.0...4.1.1)

---
updated-dependencies:
- dependency-name: js-yaml
  dependency-version: 4.1.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #105 from IlyaKozak/dependabot/npm_and_yarn/js-yaml-4.1.1

build(deps): bump js-yaml from 4.1.0 to 4.1.1

dependabot[bot] committed:
build(deps): bump next from 14.2.32 to 14.2.35

Bumps [next](https://github.com/vercel/next.js) from 14.2.32 to 14.2.35.
- [Release notes](https://github.com/vercel/next.js/releases)
- [Changelog](https://github.com/vercel/next.js/blob/canary/release.js)
- [Commits](https://github.com/vercel/next.js/compare/v14.2.32...v14.2.35)

---
updated-dependencies:
- dependency-name: next
  dependency-version: 14.2.35
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #106 from IlyaKozak/dependabot/npm_and_yarn/next-14.2.35

build(deps): bump next from 14.2.32 to 14.2.35

dependabot[bot] committed:
build(deps): bump eslint, @typescript-eslint/eslint-plugin and eslint-config-next

Bumps [eslint](https://github.com/eslint/eslint), [@typescript-eslint/eslint-plugin](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/eslint-plugin) and [eslint-config-next](https://github.com/vercel/next.js/tree/HEAD/packages/eslint-config-next). These dependencies needed to be updated together.

Updates `eslint` from 8.39.0 to 9.39.2
- [Release notes](https://github.com/eslint/eslint/releases)
- [Commits](https://github.com/eslint/eslint/compare/v8.39.0...v9.39.2)

Updates `@typescript-eslint/eslint-plugin` from 5.59.2 to 8.54.0
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v8.54.0/packages/eslint-plugin)

Updates `eslint-config-next` from 13.3.1 to 16.1.6
- [Release notes](https://github.com/vercel/next.js/releases)
- [Changelog](https://github.com/vercel/next.js/blob/canary/release.js)
- [Commits](https://github.com/vercel/next.js/commits/v16.1.6/packages/eslint-config-next)

---
updated-dependencies:
- dependency-name: eslint
  dependency-version: 9.39.2
  dependency-type: direct:production
- dependency-name: "@typescript-eslint/eslint-plugin"
  dependency-version: 8.54.0
  dependency-type: direct:production
- dependency-name: eslint-config-next
  dependency-version: 16.1.6
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #107 from IlyaKozak/dependabot/npm_and_yarn/multi-31f74126ca

build(deps): bump eslint, @typescript-eslint/eslint-plugin and eslint-config-next

dependabot[bot] committed:
build(deps): bump next from 14.2.35 to 16.1.5

Bumps [next](https://github.com/vercel/next.js) from 14.2.35 to 16.1.5.
- [Release notes](https://github.com/vercel/next.js/releases)
- [Changelog](https://github.com/vercel/next.js/blob/canary/release.js)
- [Commits](https://github.com/vercel/next.js/compare/v14.2.35...v16.1.5)

---
updated-dependencies:
- dependency-name: next
  dependency-version: 16.1.5
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #108 from IlyaKozak/dependabot/npm_and_yarn/next-16.1.5

build(deps): bump next from 14.2.35 to 16.1.5

dependabot[bot] committed:
build(deps): bump ajv from 6.12.6 to 6.14.0

Bumps [ajv](https://github.com/ajv-validator/ajv) from 6.12.6 to 6.14.0.
- [Release notes](https://github.com/ajv-validator/ajv/releases)
- [Commits](https://github.com/ajv-validator/ajv/compare/v6.12.6...v6.14.0)

---
updated-dependencies:
- dependency-name: ajv
  dependency-version: 6.14.0
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #109 from IlyaKozak/dependabot/npm_and_yarn/ajv-6.14.0

build(deps): bump ajv from 6.12.6 to 6.14.0

dependabot[bot] committed:
build(deps): bump flatted from 3.3.3 to 3.4.1

Bumps [flatted](https://github.com/WebReflection/flatted) from 3.3.3 to 3.4.1.
- [Commits](https://github.com/WebReflection/flatted/compare/v3.3.3...v3.4.1)

---
updated-dependencies:
- dependency-name: flatted
  dependency-version: 3.4.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #110 from IlyaKozak/dependabot/npm_and_yarn/flatted-3.4.1

build(deps): bump flatted from 3.3.3 to 3.4.1

dependabot[bot] committed:
build(deps): bump minimatch

Bumps  and [minimatch](https://github.com/isaacs/minimatch). These dependencies needed to be updated together.

Updates `minimatch` from 3.1.2 to 3.1.5
- [Changelog](https://github.com/isaacs/minimatch/blob/main/changelog.md)
- [Commits](https://github.com/isaacs/minimatch/compare/v3.1.2...v3.1.5)

Updates `minimatch` from 9.0.5 to 9.0.9
- [Changelog](https://github.com/isaacs/minimatch/blob/main/changelog.md)
- [Commits](https://github.com/isaacs/minimatch/compare/v3.1.2...v3.1.5)

---
updated-dependencies:
- dependency-name: minimatch
  dependency-version: 3.1.5
  dependency-type: indirect
- dependency-name: minimatch
  dependency-version: 9.0.9
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #111 from IlyaKozak/dependabot/npm_and_yarn/multi-770cfcd984

build(deps): bump minimatch

dependabot[bot] committed:
build(deps): bump next from 16.1.5 to 16.2.11

Bumps [next](https://github.com/vercel/next.js) from 16.1.5 to 16.2.11.
- [Release notes](https://github.com/vercel/next.js/releases)
- [Commits](https://github.com/vercel/next.js/compare/v16.1.5...v16.2.11)

---
updated-dependencies:
- dependency-name: next
  dependency-version: 16.2.11
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #112 from IlyaKozak/dependabot/npm_and_yarn/next-16.2.11

build(deps): bump next from 16.1.5 to 16.2.11

dependabot[bot] committed:
build(deps): bump brace-expansion

Bumps  and [brace-expansion](https://github.com/juliangruber/brace-expansion). These dependencies needed to be updated together.

Updates `brace-expansion` from 1.1.12 to 1.1.18
- [Release notes](https://github.com/juliangruber/brace-expansion/releases)
- [Commits](https://github.com/juliangruber/brace-expansion/compare/v1.1.12...v1.1.18)

Updates `brace-expansion` from 2.0.2 to 2.1.4
- [Release notes](https://github.com/juliangruber/brace-expansion/releases)
- [Commits](https://github.com/juliangruber/brace-expansion/compare/v1.1.12...v1.1.18)

---
updated-dependencies:
- dependency-name: brace-expansion
  dependency-version: 1.1.18
  dependency-type: indirect
- dependency-name: brace-expansion
  dependency-version: 2.1.4
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #113 from IlyaKozak/dependabot/npm_and_yarn/multi-5a59400ff4

build(deps): bump brace-expansion

dependabot[bot] committed:
build(deps): bump nanoid from 3.3.8 to 3.3.18

Bumps [nanoid](https://github.com/ai/nanoid) from 3.3.8 to 3.3.18.
- [Release notes](https://github.com/ai/nanoid/releases)
- [Changelog](https://github.com/ai/nanoid/blob/3.3.18/CHANGELOG.md)
- [Commits](https://github.com/ai/nanoid/compare/3.3.8...3.3.18)

---
updated-dependencies:
- dependency-name: nanoid
  dependency-version: 3.3.18
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #114 from IlyaKozak/dependabot/npm_and_yarn/nanoid-3.3.18

build(deps): bump nanoid from 3.3.8 to 3.3.18

dependabot[bot] committed:
build(deps): bump websocket-driver from 0.7.4 to 0.7.5

Bumps [websocket-driver](https://github.com/faye/websocket-driver-node) from 0.7.4 to 0.7.5.
- [Changelog](https://github.com/faye/websocket-driver-node/blob/main/CHANGELOG.md)
- [Commits](https://github.com/faye/websocket-driver-node/compare/0.7.4...0.7.5)

---
updated-dependencies:
- dependency-name: websocket-driver
  dependency-version: 0.7.5
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #115 from IlyaKozak/dependabot/npm_and_yarn/websocket-driver-0.7.5

build(deps): bump websocket-driver from 0.7.4 to 0.7.5

dependabot[bot] committed:
build(deps): bump flatted from 3.4.1 to 3.4.4

Bumps [flatted](https://github.com/WebReflection/flatted) from 3.4.1 to 3.4.4.
- [Commits](https://github.com/WebReflection/flatted/compare/v3.4.1...v3.4.4)

---
updated-dependencies:
- dependency-name: flatted
  dependency-version: 3.4.4
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #116 from IlyaKozak/dependabot/npm_and_yarn/flatted-3.4.4

build(deps): bump flatted from 3.4.1 to 3.4.4

dependabot[bot] committed:
build(deps): bump @grpc/grpc-js from 1.9.15 to 1.9.16

Bumps [@grpc/grpc-js](https://github.com/grpc/grpc-node) from 1.9.15 to 1.9.16.
- [Release notes](https://github.com/grpc/grpc-node/releases)
- [Commits](https://github.com/grpc/grpc-node/compare/@grpc/grpc-js@1.9.15...@grpc/grpc-js@1.9.16)

---
updated-dependencies:
- dependency-name: "@grpc/grpc-js"
  dependency-version: 1.9.16
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #117 from IlyaKozak/dependabot/npm_and_yarn/grpc/grpc-js-1.9.16

build(deps): bump @grpc/grpc-js from 1.9.15 to 1.9.16

dependabot[bot] committed:
build(deps): bump postcss and next

Bumps [postcss](https://github.com/postcss/postcss) to 8.5.23 and updates ancestor dependency [next](https://github.com/vercel/next.js). These dependencies need to be updated together.


Updates `postcss` from 8.4.31 to 8.5.23
- [Release notes](https://github.com/postcss/postcss/releases)
- [Changelog](https://github.com/postcss/postcss/blob/main/CHANGELOG.md)
- [Commits](https://github.com/postcss/postcss/compare/8.4.31...8.5.23)

Updates `next` from 16.2.11 to 16.3.1
- [Release notes](https://github.com/vercel/next.js/releases)
- [Commits](https://github.com/vercel/next.js/compare/v16.2.11...v16.3.1)

---
updated-dependencies:
- dependency-name: postcss
  dependency-version: 8.5.23
  dependency-type: indirect
- dependency-name: next
  dependency-version: 16.3.1
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #118 from IlyaKozak/dependabot/npm_and_yarn/multi-09870eb53c

build(deps): bump postcss and next

dependabot[bot] committed:
build(deps): bump js-yaml from 4.1.1 to 4.3.1

Bumps [js-yaml](https://github.com/nodeca/js-yaml) from 4.1.1 to 4.3.1.
- [Changelog](https://github.com/nodeca/js-yaml/blob/4.3.1/CHANGELOG.md)
- [Commits](https://github.com/nodeca/js-yaml/compare/4.1.1...4.3.1)

---
updated-dependencies:
- dependency-name: js-yaml
  dependency-version: 4.3.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

IlyaKozak committed:
Merge pull request #119 from IlyaKozak/dependabot/npm_and_yarn/js-yaml-4.3.1

build(deps): bump js-yaml from 4.1.1 to 4.3.1
